.neuro/live.yml

```diff
@@ -7,7 +7,7 @@ defaults:
   life_span: 1d
   env:
     # Those env vars will be assigned to every job in this flow
-    PACHY_URI: pachd.pachyderm.svc.cluster.local:650
+    PACHY_URI: pachd.pachyderm.svc.cluster.local:30650
```

Comment:
we access it by `NodePort`?

Reply:
pachd is still exposed via LoadBalancer, we just access it within the cluster (to avoided the use of public domain name such as `pachyderm.sandbox-delloite.org.neu.ro`)